packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -900,6 +901,63 @@ export function createLodestarMetrics(
           help: "Total number of InsertOutcome as a result of adding an aggregated attestation from api in the pool",
           labelNames: ["insertOutcome"],
         }),
+        packedAttestations: {
+          committeeBits: register.gauge<{index: number}>({
```

Comment:
maybe should rename to `committeeCount` or rename metric below to `aggregationBits`